src/tools/rustfmt/src/expr.rs

```diff
@@ -1852,7 +1852,7 @@ fn rewrite_unary_op(
 }
 
 pub(crate) enum RhsAssignKind<'ast> {
-    Expr(&'ast ast::ExprKind, Span),
+    Expr(&'ast ast::ExprKind),
```

Comment:
Please revert these rustfmt changes, and add attributes as and when necessary to avoid the warning; we very much want to have the span as part of this variant even though it's not being used just yet.